components/HorizontalFeed/Writing.tsx

```
@@ -0,0 +1,42 @@
+      setIsLoading(false);
+    };
+    if (fileUrl) getText();
+  }, [description, fileUrl]);
+
+  if (isLoading) return <Skeleton className="min-h-[200px] size-full" />;
+  const isOverflowed = height > 206;
+  return (
+    <div className="size-full relative bg-grey-eggshell">
+      <div
+        className="bg-grey-eggshell p-2 text-md !normal-case text-left"
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)